torch/csrc/api/include/torch/nn/options/loss.h

```diff
@@ -223,5 +223,17 @@ struct TORCH_API PoissonNLLLossOptions {
   TORCH_ARG(reduction_t, reduction) = torch::kMean;
 };
 
+// ============================================================================
+
+/// Options for MarginRankingLoss functional and module.
+struct TORCH_API MarginRankingLossOptions {
+  typedef c10::variant<enumtype::kNone, enumtype::kMean, enumtype::kSum> reduction_t;
+
+  /// Has a default value of `0`.
+  TORCH_ARG(float, margin) = 0;
```

Comment:
I believe we should use
```suggestion
  TORCH_ARG(double, margin) = 0;
```
because `float` in Python is always 64-bit (there are a few other places in the C++ API where we use `TORCH_ARG(float, ...)`, and we will need to change them as well :/ (Not part of this PR))